src/CancelledException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace M6Web\Tornado;
```

Comment:
Can you add a folder/namespace dedicated to exceptions? I think we will add more of them in the future (related to https://github.com/M6Web/Tornado/issues/36 )